admin_guide/limits.adoc

```diff
@@ -194,7 +194,7 @@ CPU constraint.
 If a configuration defines a `maxLimitRequestRatio` value, then any new
 containers must have both a request and limit value. Additionally,
 {product-title} calculates a limit to request ratio by dividing the limit by the
-request.
+request. Please note that this value should be a non-negative integral value greater than 1.
```

Comment:
@ravisantoshgudimetla Do we need _integral_ here? _...a non-negative value greater than 1._

Reply:
Yes integral is needed. Please look at bug.

Reply:
@ravisantoshgudimetla I was uncomfortable with _intregal_ because the more common definition of the word is not related to integers.
But, a quick Google search seems to show that _non-negative integral value_ is a common construct. 
I see _value_ used twice in the sentence now. How about:
_Please note that this value should be a non-negative integer greater than 1._